### PR TITLE
UI: Change path and ids from date now to uuid

### PR DESCRIPTION
**Pull request by kiannaquach**
- [x]  Enterprise tests pass locally

### CODE REVIEW

File: ui/tests/acceptance/enterprise-kmip-test.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ module('Acceptance | Enterprise | KMIP secrets', function (hooks) {
 
   test('it can create a role', async function (assert) {
     // moving create scope here to help with flaky test
-    const path = await mount();
+    const path = await mount(this);
```
Comment (kiannaquach) [inline ui/tests/acceptance/enterprise-kmip-test.js:213]: I'm not sure if this makes a difference, but I noticed we invoke mount with this in the [`create a scope` test](https://github.com/hashicorp/vault/blob/bd23a60c99e35dbc9d77d6cadef89bb3315cfdcd/ui/tests/acceptance/enterprise-kmip-test.js#L172)

Comment (hashishaw) [reply ui/tests/acceptance/enterprise-kmip-test.js:213]: The mount function only accepts a boolean attribute, so I'm not sure we should pass `this` in either of them 🤔

**Review by hashishaw — APPROVED** [review]
Adding UUID's is a great step. I think another improvement would be to clean up the mount after each test

File: ui/tests/acceptance/enterprise-kmip-test.js (hunk 1 above)
Comment (kiannaquach) [reply ui/tests/acceptance/enterprise-kmip-test.js:213]: Yeah I wasn't sure why that was added. I'll remove it from both

**Pull request merged**